sigs.yaml

```diff
@@ -3382,6 +3387,7 @@ sigs:
     - github: marosset
       name: Mark Rossetti
       company: Microsoft
+      email: marosset@microsoft.com
```

Comment:
confirmed with @marosset in slack

aside from marosset and aojea, all others are just filling in empty fields to match existing entries for other groups

ideally we would dedupe the person entries entirely, but that's a larger PR, and they're usually duplicated at most a few times ...